packages/fxa-auth-server/lib/routes/totp.js

```diff
@@ -130,6 +131,16 @@ module.exports = (log, db, mailer, customs, config, glean, profileClient) => {
         auth: {
           strategy: 'sessionToken',
         },
+        validate: {
+          payload: isA.object({
+            code: isA
+              .string()
+              .max(32)
+              .regex(validators.DIGITS)
+              .optional()
```

Comment:
To keep this backwards compatible, I made the code optional. However that means that for now there's no enforcement of providing a code in order to remove TOTP. Perhaps there should also be check to see if the account has a back up phone, and if it does an SMS must be provided. This was not in the ACs for the ticket though...